app/controllers/application_controller.rb

```diff
@@ -3,6 +3,9 @@ class ApplicationController < ActionController::Base
   before_action :set_cart
   helper_method :current_customer?
   helper_method :current_customer
+  helper_method :current_admin?
+  helper_method :current_admin
```

Comment:
I am wondering if we need line 5 and line 7 as helper methods, since we have the Boolean versions available.